Petar_Nedyalkov/week2/task2.cpp

```diff
@@ -0,0 +1,30 @@
+#include <iostream>
+#include <fstream>
+
+int countCertainDigits(std::ifstream& file)
+{
+	int count = 0;
+	while (!file.eof())
+	{
+		if (file.get() != 'a' || file.get() != 'd') count++;
+	}
+
+	file.close();
```

Comment:
Files are better closed in the same scope / function in which they got opened :))